app/src/main/res/values/strings.xml

```diff
@@ -285,6 +285,7 @@
     <string name="no_parcels">No parcels added</string>
     <string name="guide_part_one">Press</string>
     <string name="guide_part_two">to add</string>
+    <string name="reference_to_database_is_null_cannot_add_items">Reference to database is null, cannot add items!</string>
```

Comment:
"is null" -> "is missing"

Reply:
https://github.com/norkator/paketin-seuranta/pull/32/commits/822733aae89e1a68b4778c0704c5b04009ae428b